pyrobosim/pyrobosim/core/world.py

```diff
@@ -624,19 +626,44 @@ def remove_all_objects(self, restart_numbering=True):
         if restart_numbering:
             self.object_instance_counts = {}
 
-    def update_bounds(self):
+    def update_bounds(self, entity, remove=False):
```

Comment:
Missed this before, but can you add docstrings for the added params to this function?